freetextresponse/freetextresponse.py

```diff
@@ -298,10 +297,13 @@ def student_view(self, context={}):
                 'other_responses': self.get_other_answers(),
             }
         )
-        template = get_template('freetextresponse_view.html')
+        template = self.loader.render_django_template(
+            'templates/freetextresponse_view.html',
+            context=Context(context),
+            i18n_service=self.runtime.service(self, 'i18n'),
```

Comment:
Hmm, I'm getting an error on our Ginkgo based devstack:

![screen shot 2018-12-10 at 11 50 35](https://user-images.githubusercontent.com/3364609/49757669-d59adc00-fc71-11e8-9b3b-372ff321e34f.png)

What version of xblock-utils are you running in your Ficus devstack? Our Ginkgo stacks are running v1.0.5, and that argument seems to have been added in xblock-utils v1.1.0? (https://github.com/edx/xblock-utils/commit/ba54084a572a618b9248236403c1200f32d4f889)

Reply:
@caesar2164 Looks like I already have `xblock-utils==1.1.1` installed already. We have a couple of other PRs needing i18n and therefore using `xblock-utils==1.1.1`.

How do you suggest moving on with this issue? Should we require updating the xblock-utils or doing a nasty patch to avoid breaking the XBlock on pre-Hawthorn releases.

Reply:
@caesar2164 Are you still getting this issue or has it been resolved?